docs/sources/write/links/_index.md

```diff
@@ -35,7 +35,27 @@ Although these other types of links still function, replace them with one of the
 
 ## Link from source content that's reused as multiple pages
 
-Use `ref` URIs.
+Use a full URL and ensure the mount target has `mount_links: true` in its front matter.
```

Comment:
I'm not clear on what file the target would be. Maybe using the alerting example below, indicate what the target would be here, and where specifically one would check this. If this isn't really something you want every writer checking then maybe this should be in more of a setup section?